packages/app/src/pages/endpoints/[endpointId]/_/content/types/table/index.tsx

```diff
@@ -174,3 +205,143 @@ const Operations: React.FC<OperationsProps> = ({
     </>
   );
 };
+
+const RowObject: React.FC<
+  BaseProps & { rowObject: Data; columns: TableColumn[]; close: () => void }
+> = ({ on, rowObject, columns, close }) => {
+  return (
+    <div className="flex flex-col h-full w-full">
+      <div className="px-10 pt-10">
+        <div className="text-right">
+          <Button
+            variant="text"
+            on={COLOR_SYSTEM.SURFACE}
+            Icon={XIcon}
+            onClick={close}
+          />
+        </div>
+      </div>
+      <div className="px-10 pb-10 grow overflow-scroll">
+        <div>
```

Comment:
Same as commended above, It seems to be unnecessary.

Reply:
I will avoid unnecessary divs.